flight_phase_files/HALO_RF13_20200213_info.yaml

```diff
@@ -24,59 +24,112 @@ remarks:
 - During the whole flight precipitating cells were observed.
 - On the way to NTAS a few cold pools were observed with shallow sugar-type convection.
 segments:
-- kind: circle
+- kinds:
+  - circle
   name: c1
-  start: 2020-02-13 08:20:47
-  end: 2020-02-13 09:16:21
-- kind: circle_break
+  irregularities: []
+  segment_id: HALO-0213_c1
+  start: 2020-02-13 08:19:47
+  end: 2020-02-13 09:16:39
+  good_dropsondes: 11
+- kinds:
+  - circle_break
   name: cb1to2
-  start: 2020-02-13 09:16:21
-  end: 2020-02-13 09:29:33
-- kind: circle
+  irregularities: []
+  segment_id: HALO-0213_cb1
+  start: 2020-02-13 09:16:39
+  end: 2020-02-13 09:28:33
+- kinds:
+  - circle
   name: c2
-  start: 2020-02-13 09:29:33
-  end: 2020-02-13 10:19:35
-- kind: circle_break
+  irregularities:
+  - Time to first sonde 61 sec.
+  segment_id: HALO-0213_c2
+  start: 2020-02-13 09:28:33
+  end: 2020-02-13 10:26:08
+  good_dropsondes: 12
+- kinds:
+  - circle_break
   name: cb2to3
-  start: 2020-02-13 10:19:35
-  end: 2020-02-13 10:35:00
-- kind: circle
+  irregularities: []
+  segment_id: HALO-0213_cb2
+  start: 2020-02-13 10:26:08
+  end: 2020-02-13 10:34:01
+- kinds:
+  - circle
   name: c3
-  start: 2020-02-13 10:35:00
-  end: 2020-02-13 11:27:10
-- kind: straight_leg
+  irregularities:
+  - 'Circle is only <xarray.DataArray ''heading'' ()>
+
+    array(362.58790588) deg'
```

Comment:
In jupyter notebook put “.values” after call to heading-variable to avoid <xarray.DataArray …> output. If the output string still says “array(362….) “, use float(heading-variable.values).